### PR TITLE
fix: updated businesses card image to include

### PR DESCRIPTION
**Pull request by Benrobo**
updated business listing, business profile page to use cropped img if available otherwise switch to default

### CODE REVIEW

File: src/app/biz/[loc]/[business_id]/page.tsx [hunk 1]
```diff
@@ -131,7 +131,7 @@ export default async function BusinessPage({ params }: BizPageProps) {
           {/* business image */}
           <FlexRowStart className="w-full mt-[10px]">
             <img
-              src={constructBizImgUrl(businessDetails?.logoUrl!)}
+              src={constructBizImgUrl(businessDetails?.croppedImageUrl!)}
```
Comment (osxar) [inline src/app/biz/[loc]/[business_id]/page.tsx:134]: @Benrobo given that the croppedImageUrl is a new feature, our currently registered businesses on both dev and prod, doesn't have the url. Please setup a check that does: 1) check for the croppedUrl, if it doesn't exists, then it checks the originalUrl, if that doesn't exists, then it renders the default business logo

File: src/utils/index.ts [hunk 2]
```diff
@@ -94,10 +94,11 @@ export const isUrlValid = (url: string) => {
...
 };
 
-export const constructBizImgUrl = (url: string | null) => {
-  return !url
-    ? defaultImg
-    : `https://res.cloudinary.com/${CloudinaryConfig.cloudName}/image/upload/c_fill,q_500/${url}.jpg`;
+export const constructBizImgUrl = (publicId: string | null) => {
+  console.log({ publicId });
+  return !publicId
+    ? "https://res.cloudinary.com/drwt2qqf9/image/upload/v1721488956/default-img_vhxk4d.jpg"
```
Comment (osxar) [inline src/utils/index.ts:100]: @Benrobo why don't we set this up as a variable in a config file

**Review by osxar — CHANGES_REQUESTED** [review]
When I create a new business with a cropped logo, the business fails to save.

**Comment by osxar** [comment]
@Benrobo Also the default image looks stretched. Please use the new one on figma, thanks.
Screenshot 2024-08-21 at 12 11 38 AM

File: src/app/biz/[loc]/[business_id]/page.tsx (hunk 1 above)
Comment (Benrobo) [reply src/app/biz/[loc]/[business_id]/page.tsx:134]: concerning this, the image displayed on the business card should be the cropped version of that image. When the image doesn't exists or when the user doesn;t upload one, that value becomes null, which automatically get assigned to the default image in cloudinary.

Comment (osxar) [reply src/app/biz/[loc]/[business_id]/page.tsx:134]: yhea, i get that, but if we deploy this change as it is, all the business profile that we currently have will show the default business image, and we don't want that. We will have to send an email to each users telling them of the upgrade we have done to the application and requesting them to crop their image the way they would like it. When that is done, we can then revert to this implementation.

**Comment by osxar** [comment]
@Benrobo I also noticed that these pages are still using the originalUrl instead of the cropped url:

1) Similar businesses on business details page
2) My business listing page (where a business owner goes to checkout a list of his/her businesses)

**Review by osxar — APPROVED** [review]

**Pull request merged**